blockchain/chain.go

```diff
@@ -2314,6 +2361,7 @@ func New(config *Config) (*BlockChain, error) {
 		mainchainBlockCacheSize: mainchainBlockCacheSize,
 		deploymentCaches:        newThresholdCaches(params),
 	}
+	b.initializeStakeVersionCaches()
```

Comment:
As discussed on the `initializeStakeVersionCaches` func, I would really prefer this be done inside the struct to be consistent with all the other map makes.